install-maven2-dependencies-and-copy-to-maven1-repository.sh

```diff
@@ -11,64 +11,67 @@
 ## They are in alphabetical order by groupId, artifactId.
 
 # je-3.3.62.jar is available from spring plugins com.sleepycat:je:3.3.62 (April 2018?) (18.3.12 is available November 2018)
-mvn org.apache.maven.plugins:maven-dependency-plugin:2.1:get -DrepoUrl=http://repo.spring.io/plugins-release/ \
-  -Dartifact=com.sleepycat:je:3.3.62
+# je-3.3.62.jar is no longer available from spring plugins (Feb 2020), we now load it manually.
+mvn install:install-file -DgroupId=com.sleepycat -DartifactId=je \
+      -Dversion=3.3.62 \
+      -Dpackaging=jar \
+      -Dfile=./lib/je-3.3.62.jar
 mkdir -pv ~/.maven/repository/com.sleepycat/jars
-cp -v ~/.m2/repository/com/sleepycat/je/3.3.62/je-3.3.62.jar ~/.maven/repository/com.sleepycat/jars
+cp -v /c/AppDev/maven-repo/com/sleepycat/je/3.3.62/je-3.3.62.jar ~/.maven/repository/com.sleepycat/jars
```

Comment:
This new path doesn't match the default location of the Maven 2 repo, which is $USER_HOME/.m2/repository (~/.m2/repository on Mac and Linux). Is there something special going on here?

Reply:
Woops left that in there by mistake, will fix